1.7/administration/securing-your-cluster.md

```diff
@@ -47,6 +82,14 @@ only specific tasks can be scheduled here. These agent nodes have both public
 and private IP addresses and only specific ports should be open in their
 iptables firewall.
 
+By default, when using the cloud-based installers such as the AWS
+CloudFormation templates, a large number of ports are exposed to the Internet
+for the public zone. In production systems, it is unlikely that you would
+expose all of these ports. It's recommended that you close all ports except
+80 and 443 (for HTTP/HTTPS traffic) and use
```

Comment:
Even if you make everything HTTPS-only, you'll still have to redirect HTTP traffic to HTTPS (which requires port 80).